source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -120,26 +139,36 @@ namespace
     void printConnectHeader(const size_t numConnects)
     {
         printHeaderLine(numConnects, { "Connecting and disconnecting ", " times each (be patient):" });
+        addToRawFile("Connecting and disconnecting ");
     }
 
     void printCapture3DHeader(const size_t numFrames, const std::vector<Zivid::Settings> &settingsVector)
     {
         const auto filterList = makefilterList(settingsVector);
         printHeaderLine(numFrames, { "Capturing ", " 3D frames:" });
+        addToRawFile("Capturing 3D frames:");
         std::cout << "  exposure time = " << makeSettingList<Zivid::Settings::ExposureTime>(settingsVector)
                   << std::endl;
+        addToRawFile("  exposure time = " + makeSettingList<Zivid::Settings::ExposureTime>(settingsVector));
```

Comment:
You add exposure time here but you don't add it at line 171 in capture2D. Choose one, my vode goes to minimalism.

Reply:
I'll add it to both, we need to for 3D to distinguish the data